### DISCUSSION TITLE
What's best way to export trpc generated types to a standalone react native app (different developers/team)?

### DISCUSSION THREAD

**Discussion by shahidcodes** [trpc/trpc#2146] [Q&A]
Can we export types and publish it to a package and share with other team member or developer without converting the project in a mono repo or putting both back end and frontend in the same repo?

**Comment by KATT** [comment]
Yeah, you can export types of the backend as an npm lib but I'd recommend having a monorepo.

**Comment by shahidcodes** [comment]
Could you direct me to any docs which help export the types only? 

```
export type AppRouter = typeof appRouter;
```

This needs `appRouter`. I don't know how to extract appRouter type to a file.

**Reply by Dxig** [reply]
Try typescript cli.

```
tsc index.ts --declaration --emitDeclarationOnly
```
